apps/moodlenet/vars/all/main.yml

```diff
@@ -1,7 +1,8 @@
 # Application default configuration
 
-# -- route
-moodlenet_host: "moodlenet.{{ project_name}}.{{ domain_name }}"
+# -- route / urls
+moodlenet_host: "api.{{ project_name}}.{{ domain_name }}"
```

Comment:
Test are failing because of that change. I will change the test according to this change,